deploy/template.yaml

```diff
@@ -487,7 +487,7 @@ Resources:
       TargetTrackingScalingPolicyConfiguration:
         PredefinedMetricSpecification:
           PredefinedMetricType: ECSServiceAverageCPUUtilization
-        TargetValue: 60
+        TargetValue: 50
```

Comment:
We might want to tune this back up once we've got a passing test, but should be safe enough for now.

If we reach a steady-state of 50%, we're paying for twice as much capacity as we're actually using! I assume our production usage is currently well below this for now anyway?

Reply:
That makes sense, if it looks good then I will revert 👍